config/hydroshare-config.yaml

```diff
@@ -33,16 +33,16 @@ IRODS_LOCAL_MOUNT: /irods
 ### These give Nginx direct access to iRODS vaults
 ### All ROOTs must be children of DOCKER_MAP ###
 IRODS_DOCKER_MAP: /irods
-IRODS_DATA_ROOT: /irods/data
-IRODS_USER_ROOT: /irods/user
+IRODS_DATA_ROOT: /irods/data/iRODS/Vault/home/wwwHydroProxy
+IRODS_USER_ROOT: /irods/user/iRODS/Vault/home/localHydroProxy
 IRODS_CACHE_ROOT: /irods/cache
 
 ### Nginx worker process must be able to read iRODS files.
 ### These used to build the nginx docker container.
 SENDFILE_IRODS_USER: irods
 SENDFILE_IRODS_GROUP: irods
-SENDFILE_IRODS_USER_ID: 20000
-SENDFILE_IRODS_GROUP_ID: 20000
+SENDFILE_IRODS_USER_ID: 998
+SENDFILE_IRODS_GROUP_ID: 998
 
```

Comment:
@hyi made an earlier comment about hardwiring account ids I believe. is this ok here?

Reply:
That's irods UID inside data.local.org and users.local.org. If we change it, means we use other iRODS settings ===> no needs this PR